.github/workflows/main.yml

```diff
@@ -143,3 +143,34 @@ jobs:
       - name: QA (unit)
         run: |
           poetry run invoke test --no-flake
+
+  build_3_12:
```

Comment:
I've just realized this build is not defined correctly - you can re-use all the below by using the `matrix` syntax, see https://github.com/4dn-dcic/utils/blob/master/.github/workflows/main.yml#L24-L26 as an example